tools/find_image.rb

```diff
@@ -32,14 +32,14 @@ def run
           :build_type => "nightly-",
           :url => "https://openshift-release.svc.ci.openshift.org/"
         raise "missing query string" if options.query.nil?
-        doc = Nokogiri::HTML(open(options.url).read)
+        doc = Nokogiri::HTML(URI.open(options.url).read)
         links = doc.xpath("//a[contains(text(), '#{options.build_type}')]")
 
         target_links = links.map { |l|  options.url + l.attributes['href'].value }
         threads = []
         target_links.each do |link|
           threads << Thread.new(link) do |i|
-            doc = Nokogiri::HTML(open(link).read)
+            doc = Nokogiri::HTML(URI.open(link).read)
```

Comment:
@pruan-rht , Ithink the other fixes are already merged, do you want to submit only changes to this file?